src/components/Tooltip/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { ReactNode } from "react"
+import {
+  Box,
+  Center,
+  Popover,
+  PopoverArrow,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+} from "@chakra-ui/react"
+import * as utils from "../../utils/isMobile"
+
+export interface IProps {
+  content: ReactNode
+  children?: React.ReactNode
+}
+
+const Tooltip: React.FC<IProps> = ({ content, children }) => {
+  const isMobile = utils.isMobile()
+
+  return (
+    <Box title="More Info!" display="inline-block" ml={2}>
+      <Popover placement="top" trigger="hover" strategy="fixed" gutter={6}>
+        <PopoverTrigger>
+          <Center
+            as="span"
+            tabIndex={0}
+            p={1}
+            cursor="pointer"
+            color="body.base"
+            _hover={{
+              color: isMobile ? "primary.hover" : "primary.base",
+            }}
```

Comment:
Per the figma file, the `primary.hover` is marked for the `:active` state, which makes sense in mobile because you have to touch the icon in order to get the popover to appear.

Instead of trying to spend a check, I would use `_active` pseudo prop. This would be semantically correct and follows the [W3C Specifications](https://drafts.csswg.org/selectors/#the-active-pseudo).

```suggestion
            _hover={{
              color: "primary.base",
            }}
            _active={{
              color: "primary.hover",
            }}
```

@pettinarip I'm interpreting this as the `:active` color is showing onPress, and then when releasing the press it switches to the `:hover/open` color per the figma file labelling.